homeassistant/components/unifiprotect/binary_sensor.py

```diff
@@ -96,6 +98,22 @@ class ProtectBinaryEntityDescription(
         device_class=BinarySensorDeviceClass.MOTION,
         ufp_value="is_motion_detected",
         ufp_last_trip_value="motion_detected_at",
+        ufp_enabled="is_motion_sensor_enabled",
+    ),
+    ProtectBinaryEntityDescription(
+        key="alarm_sound",
+        name="Alarm Sound Detected",
+        device_class=BinarySensorDeviceClass.PROBLEM,
```

Comment:
```suggestion
        device_class=BinarySensorDeviceClass.SOUND,
```
Is `SOUND` more fitting here?

Reply:
Based on the description of the enum, I would say yes.